pkgs/by-name/le/lexbor/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "lexbor";
+  version = "2.4.0";
```

Comment:
If you want to align it with the version vendored in php src you should use https://github.com/lexbor/lexbor/commit/0eac579f